tensorflow/compiler/xla/service/gpu/tests/gpu_ftz_test.cc

```diff
@@ -76,13 +76,23 @@ class GpuFtzDisabledTest : public GpuFtzTest {
 };
 
 // Check that we emit mul.ftz.f32 when in ftz mode, and plain mul.f32 otherwise.
+//
+// On the ROCM platform the "ptx" string is not populated for the compiled
+// executable, and hence the call to CompileAdnVerifyPtx does not do the
+// "VerifyPtx" part, it merely compiles the executable
```

Comment:
Typo: CompileAndVerifyPtx.

Are we testing anything in ROCM mode here at all? Would you like to just test the compilation?

Currently the API creates a false impression of testing on ROCM which ideally would be good to avoid.

You could also have a test-local function e.g. `CompileAndOptionallyVerifyPtx`, which would verify PTX only if running on CUDA?

Reply:
in the next commit, I will rename the existing `CompileAndVerifyPtx` routine to `CompileAndOptionalyVerifyPtx`, add comments in the `gpu_codegen_test.[h,cc]` indicating why, and also remove all instances of the comment above